apps/ts-minbar-test-react/package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "@fluentui/ts-minbar-test-react",
-  "version": "1.0.0",
+  "version": "8.0.0",
   "private": true,
   "description": "Testing Fluent UI React compatibility with Typescript 3.9",
   "license": "MIT",
   "dependencies": {
     "@fluentui/react": "^8.103.0"
   },
   "scripts": {
-    "build": "just-scripts build",
+    "type-check": "tsc -p .",
     "test": "node -r @fluentui/scripts/babel/register src/index.ts"
```

Comment:
As `babel.config.js` is removed, should be this `ts-node --swc`?

Reply:
great catch, yup!